src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/ReadsPreprocessingPipeline.java

```diff
@@ -80,8 +89,12 @@ protected void setupPipeline( Pipeline pipeline ) {
         final PCollectionView<SAMFileHeader> headerSingleton = ReadsDataflowSource.getHeaderView(pipeline, readsHeader);
         final PCollection<GATKRead> initialReads = readsDataflowSource.getReadPCollection(intervals);
 
+        final OpticalDuplicateFinder finder = READ_NAME_REGEX != null ?
+            new OpticalDuplicateFinder(READ_NAME_REGEX, OPTICAL_DUPLICATE_PIXEL_DISTANCE, null) : null;
+        final PCollectionView<OpticalDuplicateFinder> finderPcolView = pipeline.apply(Create.of(finder)).apply(View.<OpticalDuplicateFinder>asSingleton());
```

Comment:
You can just pass it as a constructor argument and dataflow will serialize it for you.
@davidaadams Can you confirm?